abrechnung/sftpgws.py

```diff
@@ -114,35 +122,45 @@ async def handle_ws_connection(self, request):
 
         async with pool.acquire() as connection:
             # register the client at the db
-            client_id = await connection.fetchval(
-                "SELECT client_connected($1);",
+            connection_id = await connection.fetchval(
+                "select * from client_connected($1);",
                 self.cfg['websocket']['id']
             )
 
-            self.open_connections[client_id] = ws
+            # when too many unsent messages are in the queue, error.
+            tx_queue = asyncio.Queue(maxsize=1000)
+
+            self.tx_queues[connection_id] = tx_queue
+            asyncio.create_task(self.tx_task(ws, tx_queue))
```

Comment:
we should track this task somehow, and we should add a future-done-callback to somehow handle exceptions in the task (e.g. log them)